packages/twenty-ui/tsconfig.json

```diff
@@ -10,7 +10,8 @@
     "types": ["node"],
     "outDir": "../../.cache/tsc",
     "paths": {
-      "@ui/*": ["packages/twenty-ui/src/*"]
+      "@ui/*": ["packages/twenty-ui/src/*"],
+      "twenty-shared": ["packages/twenty-shared/dist"]
```

Comment:
``` suggestion
"twenty-shared": ["packages/twenty-shared/src/*"]
```
Please target the barrel file here `packages/twenty-shared/src/index.ts` instead of targeting dist